Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM cgr.dev/chainguard/static:latest
 ARG TARGETARCH
 
-USER nonroot:nonroot
+USER 65532:65532
 
 COPY --chown=nonroot:nonroot "build/zarf-linux-$TARGETARCH" /zarf
```

Comment:
Thanks for the PR! Just for readability would request the `chown` be made the same UID with a comment left explaining what the UID refers to, otherwise LGTM.